bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt

```diff
@@ -127,16 +130,46 @@ object GiniBank {
      */
     fun setCaptureConfiguration(context: Context, captureConfiguration: CaptureConfiguration) {
         GiniBank.captureConfiguration = captureConfiguration
-        GiniCapture.newInstance(context)
-            .applyConfiguration(captureConfiguration)
-            .build()
+        GiniCapture.newInstance(context).applyConfiguration(captureConfiguration).build()
         giniCapture = GiniCapture.getInstance()
     }
 
+
+    /**
+     * Provides transfer summary to Gini.
+     *
+     * Please provide the required transfer summary to improve the future extraction accuracy.
```

Comment:
[Suggestion] we have 4 sentences in row, which are starting from `Please`. I propose to keep it till `Please follow the recommendations below` and remove other please after.

Reply:
@zladzeyka Just to be on the same page, you mean I delete the `please`s after the first one?